docs/source-1.0/spec/aws/error-rename-simple.rst.template

```diff
@@ -0,0 +1,7 @@
+--------------------
+Error shape renaming
+--------------------
+
+By default, Smithy permits renaming shapes to disambiguate shape ID conflicts in
+the :ref:`service closure <service-closure>` via the ``rename`` property. However, services using this protocol are
+not allowed to rename error shapes (shapes with :ref:`error trait <error-trait>` applied).
```

Comment:
This is the simplified version for protocols that send the shape ID over wire. It will be removed later when AWS SDKs have been verified to support error shape rename.